### PR TITLE
🧁  Add home page to corona diaries 🧸

### PR DESCRIPTION
**Pull request by lisamittmann**
- add `Homepage.js` with background and styled buttons
- adjust `App.js`

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -5,11 +5,15 @@ import RecipeOverview from "./pages/RecipeOverview";
...
 import CoronaOverview from "./pages/CoronaOverview";
+import Homepage from "./pages/Homepage";
 
 function App() {
     return (
         <div>
             <Switch>
+                <Route exact path="/">
+                    <Homepage/>
+                </Route>
```
Comment (jamarob) [inline frontend/src/App.js]: short'n'sweet 
```suggestion
                <Route exact path="/" component={Homepage}/>
```

File: frontend/src/App.js [hunk 2]
```diff
@@ -5,11 +5,15 @@ import RecipeOverview from "./pages/RecipeOverview";
 import NavigationBar from "./components/NavigationComponents/NavigationBar";
 import PageLayoutWithNavigation from "./components/NavigationComponents/PageLayoutWithNavigation";
 import CoronaOverview from "./pages/CoronaOverview";
+import Homepage from "./pages/Homepage";
 
 function App() {
     return (
         <div>
```
Comment (jamarob) [inline frontend/src/App.js]: Why the div? I think this can go.

File: frontend/src/pages/Homepage.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components/macro'
+import {Link} from "react-router-dom";
+
+export default function Homepage() {
+    return (
+        <HomeLayout>
+            <h1>Teddys Corona Diaries</h1>
+            <HomeButton to="/recipes">REZEPTE</HomeButton>
```
Comment (jamarob) [inline frontend/src/pages/Homepage.js:8]: 👍 for styling a link as a button!

File: frontend/src/pages/Homepage.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    @media (min-width: 550px) {
+      font-size: 40px;
+    }
+    @media (max-width: 549px) {
+      font-size: 28px;
+    }
+  }
+`
+
+const buttonStyle = `
```
Comment (jamarob) [inline frontend/src/pages/Homepage.js:41]: you may have to overwrite `:visited` as well, or the color may change for the anchor after it is clicked.

**Review by jamarob — APPROVED** [review]
LGTM 🧸

File: frontend/src/App.js (hunk 1 above)
Comment (lisamittmann) [reply frontend/src/App.js]: 🦑  love that, done

**Pull request merged**